### PR TITLE
fix and refactor python examples

### PR DESCRIPTION
**Pull request by joein**

### CODE REVIEW

File: qdrant-landing/content/documentation/tutorials/create-snapshot.md [hunk 1]
```diff
@@ -114,10 +114,10 @@ Calculating the embeddings is usually a bottleneck of the vector search pipeline
 ```python
 ids, vectors, payloads = [], [], []
 for payload in dataset:
-    id = payload.pop("id")
+    id_ = payload.pop("id")
```
Comment (Anush008) [inline qdrant-landing/content/documentation/tutorials/create-snapshot.md:117]: `id()` thanks you.

**Review by Anush008 — APPROVED** [review]
🫡

File: qdrant-landing/content/documentation/concepts/explore.md [hunk 2]
```diff
@@ -455,12 +455,11 @@ POST /collections/{collection_name}/points/recommend/batch
...
 ```python
-from qdrant_client import QdrantClient
-from qdrant_client.http import models
+from qdrant_client import QdrantClient, models
 
-client = QdrantClient("localhost", port=6333)
+client = QdrantClient(url="http://localhost:6333")
 
-filter = models.Filter(
+recommend_filter = models.Filter(
```
[the remark this answers is not in the document]
Comment (joein) [reply qdrant-landing/content/documentation/concepts/explore.md:462]: `filter` is a builtin python function, overriding those is not a good practice

[the remark this answers is not in the document]
Comment (coszio) [reply qdrant-landing/content/documentation/concepts/explore.md:462]: Ah right! That question had bugged me before. I know we already have `scroll_filter` argument, but would it be better to use `filter_` for these variables?

[the remark this answers is not in the document]
Comment (joein) [reply qdrant-landing/content/documentation/concepts/explore.md:462]: I preferred using more eloquent name, `filter_` seems a little bit ugly for me

[the remark this answers is not in the document]
Comment (coszio) [reply qdrant-landing/content/documentation/concepts/explore.md:462]: we do the same for other builtins like `id_`, what's the difference?

**Review by coszio — APPROVED** [review]
Approving since there are more good things here than the taste difference

**Pull request merged**